app/middleware/tracker_middleware.rb

```diff
@@ -0,0 +1,27 @@
+class TrackerMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    dup._call(env)
```

Comment:
Why the .`dup`?

Reply:
I thought that was a good idea after reading https://ieftimov.com/writing-rails-middleware see `Thread Safety`